index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="logo_placeholder">
+            <p class="logo white">
+                John Doe
+            </p>
+        </div>
+        <nav>
+            <div class="bars">
+                <img src="icons/bars.png" alt="nav bars">
+            </div>
+        </nav>
+    </header>
+    <section class="headline_section">
+        <div class="primary_text">
+            Hey There. I’m Alex
+        </div>
+        <div class="supporting_text">
+            I’am a Software Developer
+        </div>
+        <div class="description white">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </div>
+        <div class="social_media">
+            <div class="vector">
+                <img src="icons/github.png" alt="github">
+            </div>
+            <div class="vector">
+                <img src="icons/linkedin.png" alt="linkedin">
+            </div>
+            <div class="vector">
+                <img src="icons/angelist.png" alt="angelist">
+            </div>
+            <div class="vector">
+                <img src="icons/twitter.png" alt="vector">
+            </div>
+            <div class="vector">
+                <img src="icons/medium.png" alt="medium">
+            </div>
+        </div>
```

Comment:
- [ ] Kindly use `ul` tag to wrap the whole links
- [ ] Kindly wrap each of the `img` in an `a` tag and add `href` to the `a` tag. learn more on how to use it [here](https://www.w3schools.com/tags/tag_a.asp)